src/test/resources/test_data/calculation_breakdown_response/custom-examples/crs-2047-sds-40-labels-5.json

```diff
@@ -55,16 +55,15 @@
       "unadjustedDate": "2027-04-22"
     },
     "CRD": {
-      "rules": ["SDS_STANDARD_RELEASE_APPLIES"],
+      "rules": [],
```

Comment:
Interesting this is no longer being added.
I get this isn't actually what this test is testing, but what's changed here.

Reply:
So this one, there is a SDS standard release sentence with a release date in 2025 and a concurrent early release sentence with a release date before the tranches. 

So I think its correct that we wouldn't have any SDS40 related breakdown here. We don't go in the SDS defaults

In fact the ticket this came from [2047](https://dsdmoj.atlassian.net/browse/CRS-2047) says that example 5 shouldn't display any labels!

Reply:
Cool thanks for checking